src/java/org/apache/cassandra/index/accord/RangeMemoryIndex.java

```diff
@@ -158,14 +160,28 @@ public synchronized NavigableSet<ByteBuffer> search(int storeId, TableId tableId
         return pks;
     }
 
-    private TreeMap<Range, Set<DecoratedKey>> search(RangeTree<byte[], Range, DecoratedKey> tokensToPks, byte[] start, byte[] end)
+    public TreeMap<Range, Set<DecoratedKey>> search(RangeTree<byte[], Range, DecoratedKey> tokensToPks, byte[] start, byte[] end)
     {
 
         TreeMap<Range, Set<DecoratedKey>> matches = new TreeMap<>();
         tokensToPks.search(new Range(start, end), e -> matches.computeIfAbsent(e.getKey(), ignore -> new HashSet<>()).add(e.getValue()));
         return matches;
     }
 
+    public synchronized NavigableSet<ByteBuffer> search(int storeId, TableId tableId, byte[] key)
+    {
+        var rangesToPks = map.get(new Group(storeId, tableId));
```

Comment:
```suggestion
        RangeTree<byte[], Range, DecoratedKey> rangesToPks = map.get(new Group(storeId, tableId));
```

since there is a desire to ban `var` this should be `RangeTree<byte[], Range, DecoratedKey>`